.github/ISSUE_TEMPLATE/92_tech_open-review.md

```diff
@@ -0,0 +1,31 @@
+---
+name: "[Tech Chair -> Open Practices Chair] Open Materials Review"
+about: Review for Open Materials Chair to fill out
+title: "[REVIEW] Open Materials Review"
+labels: review
+assignees: ''
+---
+
+<!--
+## DO NOT EDIT THIS FILE OUTSIDE OF THE journalovi/jovi-workflows REPOSITORY
+##
+## This file is automatically updated in all repositories within the journalovi
+## Github organization whenever the version in journalovi/jovi-workflows is
+## changed, so any other edits will be overwritten. To update this file, make
+## a commit or pull request at https://github.com/journalovi/jovi-workflows
+-->
+
+### Instructions for Open Practices Chair
+- If there is anything in the list below that you are unsure of, you can reply to this issue (after it has been created) and tag the responsible organizers. Organizers' Github IDs are listed on [People page](https://www.journalovi.org/people.html).
+- If anything in the below list is missing (e.g. required paper sections, supplemental materials, etc), tag the authors on this issue and ask them to add those materials.
```

Comment:
When the Open Practices Chair work on this issue, there may already be several people added to the repo. Is there a way for the Open Practices Chair to know who are the author(s) that they should tag?

Reply:
Good point. We should probably have a list of author, OC, and AE github IDs somewhere, maybe top of the README

Reply:
Initial tech setup now has the Tech Chair add author IDs to the top of README.md, and I've updated the instructions here to point Open Practices chairs there to find the IDs.